builtin/logical/transit/path_encrypt.go

```diff
@@ -126,7 +126,7 @@ to the min_encryption_version configured on the key.`,
 			"partial_failure_response_code": {
 				Type: framework.TypeInt,
 				Description: `
-Ordinarily, if a batch item fails to encrypt due to a bad input, but other batch items succeed, 
+Ordinarily, if a batch item fails to encrypt due to a bad input, but other batch items succeed,
```

Comment:
I'd revert this change, I think this will be reflowed by the `path-help` endpoint and the newline removed, so you'll need the space in here to avoid the comma from going into the next word.

Reply:
Yes, sure. This was automatically done by the editor and I missed it when staging.